provenance.md

```diff
@@ -0,0 +1,24 @@
+# Provenance
+[See issues](https://github.com/OCR-D/spec/issues/11)
+All provenance have to be stored in files. 
+There's provenance at the page level, but there's also provenance at the document level.
+Because these files can become very large, they are also stored as references in the fileGrp section.
+
+## Format
+Not defined yet
+
+### Input/Output
```

Comment:
Since a step in the workflow will be a callout to a tool, I'd suggest basing the input on the concrete call, with defaults taken from the ocrd-tool.json.